### ISSUE TITLE
Using render function for Rmd

### ISSUE THREAD

**Issue by SivuyileNzimeni** [datawookie/emayili#80]
Attempted to render Rmd into an email. The error code appears to the referring to reading in the file, see the error message below:
**Error: '/private/var/folders/m5/s2kx6dpn5kd0pnmmg68z3ctc0000gn/T/RtmpoHfuAc/2021_Vehicle_Prices_2021-09-19.csv' does not exist.** 
I think it may be an issue with the temporary directory and document directory. I have added the files utilised on the link below.
[Implememted Example](https://github.com/SivuyileNzimeni/exploring_emayili)

**Comment by datawookie** [comment]
Aha! I think I see the problem here. I'm making a copy of the `.Rmd` file to a temporary location and then the render happens in that location... so it doesn't see your CSV. I need to make the temporary file in the _same_ location as the original `.Rmd` file. Let me sort this out and get back to you.

Thanks for the bug report, @SivuyileNzimeni.

**Comment by datawookie** [comment]
Hi @SivuyileNzimeni, I've pushed a patch. Can you please test?

```r
remotes::install_github("datawookie/emayili", ref = "962903b569a1e6717fad9fc0151060c452f82968")
```

Thanks, Andrew.

**Comment by SivuyileNzimeni** [comment]
Great news Andrew,
`remotes::install:_github("datawookie/emayili", ref= "f2008e21a5d55700ad683f64349a3b6f8f538d51")` appears to have done the trick. 
The file renders as expected.

**Comment by datawookie** [comment]
These changes are available in [v0.5.2](https://github.com/datawookie/emayili/releases/tag/v0.5.2).
